DayrSuryan/DSEthiop15.xml

```diff
@@ -136,97 +140,132 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                              <foreign xml:lang="gez">ወ<persName ref="PRS12977WaldaTaklaHaymanot">ተክለ፡ ሃይማኖት፡</persName>
                               </foreign> added in green below the text block.</desc>
                         </item>
-                        
+
                         <item xml:id="e2">
                            <locus target="#3r #9r"/>
                            <desc>An omitted character has been added in very small script above the line.</desc>
                         </item>
-                        
+
                         <item xml:id="e3">
                            <locus target="#6v #7r #7vb"/>
                            <desc>Omitted characters have been added by the same hand above the line.</desc>
                         </item>
-                        
+
                         <item xml:id="e4">
                            <locus target="#4ra #6v"/>
                            <desc>A word added by mistake has been expunctuated in red and black.</desc>
                         </item>
-                        
+
                         <item xml:id="e5">
                            <locus target="#5ra"/>
                            <desc>A mistaken character has been marked by black dashes above and below and erased.</desc>
                         </item>
-                        
+
                         <item xml:id="e6">
                            <locus target="#5rb"/>
                            <desc>A mistaken character has been marked by red dashes above and below and erased.</desc>
                         </item>
-                        
+
                         <item xml:id="e7">
                            <locus target="#5va"/>
                            <desc>A mistaken character has been expunctuated in red.</desc>
                         </item>
-                        
+
                         <item xml:id="e8">
                            <locus target="#5vb #7r #7vb #8rb #8va #8vb #9r"/>
                            <desc>Mistaken characters have been marked by a black line above and below.</desc>
                         </item>
-                        
+
                         <item xml:id="e9">
                            <locus target="#7r"/>
                            <desc>A mistaken character has been encircled in red.</desc>
                         </item>
-                        
+
                         <item xml:id="e10">
                            <locus target="#7va"/>
                            <desc>A mistaken character has been crossed out in red.</desc>
                         </item>
-                        
+
                         <item xml:id="e11">
                            <locus target="#4"/>
                            <desc>The parchment has been mended.</desc>
                         </item>
                      </list>
```

Comment:
This can be deleted, it is included above

Reply:
Done.